### PR TITLE
make nav bar and footer responsive for mobile and laptops

### PR DESCRIPTION
**Pull request by rohit123-rawat**
## Description

- Make nav bar and footer responsive for small and large devices

### CODE REVIEW

File: src/Components/Footer/footer.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  -webkit-text-fill-color: rgb(28, 43, 249);
+  margin-top: 1rem;
+  padding: 1rem;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height:10%;
+  /* a {
+    color: white;
```
Comment (LavishRawat) [inline src/Components/Footer/footer.css:30]: remove commented code

File: src/Components/Footer/footer.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    } else {
+      return (
+        <Card className="footers">
+         @Thanks to visiting our site
+          <a>About us</a>
+        </Card>
+      );
+    }
+}
+export default Footer;
```
Comment (LavishRawat) [inline src/Components/Footer/footer.js:22]: add a line at the end of code

File: src/Components/Navbar/navbar.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    border-color: rgb(56, 52, 52);
+}
+
+.details {
+    margin-left: 40px;
+    margin-right: 40px;
+}
+
+/* for small devices */
+/*
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:29]: remove commented code from 29-31

File: src/Components/Navbar/navbar.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    border-radius: 180%;
+    border-color: rgb(87, 78, 78);
+}
+.align-mob {
+    width: 200px;
+}
+
+.searchArea-mob {
+    width: 160px;
+}
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.css:62]: no new line at the end of code

File: src/Components/Navbar/navbar.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+                  Add Product
+                </Nav.Link>
+              </Nav>
+            </Navbar.Collapse>
+        </Container>
+      </Navbar>
+    );
+  } else {
+    return (
+
```
Comment (LavishRawat) [inline src/Components/Navbar/navbar.js:48]: remove extra line

File: src/Components/Footer/footer.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
+.footers {
+    margin-top: 1rem;
```
Comment (Abhinegi2) [inline src/Components/Footer/footer.css:2]: can we use "px" rather than rem.

File: src/Components/Footer/footer.css [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    padding: 1rem;
+    background-color: rgb(250, 244, 244);
+    position: fixed;
+    bottom: 0;
```
Comment (Abhinegi2) [inline src/Components/Footer/footer.css:6]: remove unnecessary styling

File: src/Components/Navbar/navbar.css [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+/* 
+/* 
+ */
+
+.navbar-mob {
+    margin-left: 1px;
+}
+.logo-mob {
+    width: 40px;
+    margin-left: 0%;
```
Comment (Abhinegi2) [inline src/Components/Navbar/navbar.css:38]: !

File: src/Components/Navbar/navbar.css [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.home-mob {
+    width: fit-content;
+}
+
+.search-mob {
+    width: 20%;
+    margin-left:10px;
+    margin-right: 10pc;
```
Comment (Abhinegi2) [inline src/Components/Navbar/navbar.css:52]: pc??

File: src/Components/Navbar/navbar.js [hunk 10]
```diff
@@ -0,0 +1,84 @@
+import Container from 'react-bootstrap/Container';
```
Comment (Abhinegi2) [inline src/Components/Navbar/navbar.js:1]: can we use something like this,
import {container, form,nav,navbar.......} from 'react-bootstrap'

**Review by Abhinegi2 — APPROVED** [review]

**Pull request merged**